src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,42 @@
+.calendar {
+  width: $calendar-width;
+  padding: $padding;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+
+  @each $week-day, $index in $week-days {
+    &--start-day--#{$week-day} > :first-child {
+      margin-left: ($cell-size + $gap) * $index;
+    }
+  }
+
+  @for $last-day from 28 to 31 {
+    &--month-length--#{$last-day} > :nth-last-child(-n + #{31 - $last-day}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    width: $cell-size;
+    height: $cell-size;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #eee;
+    border: 1px solid black;
+    transition: transform 0.5s ease-in-out;
```

Comment:
Don't forget to specify to what properties you apply the transition.
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/f3918f97-6864-41ce-8ee6-3841d8a06e95)

Reply:
Good attention.